packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -619,11 +619,7 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
       return stepToAdd;
     }
   });
-
   if (addedSteps) {
-    const eventTypesToCreateReminders = activeOn.filter(
-      (activeEventType) => activeEventType && !newEventTypes.includes(activeEventType)
```

Comment:
As we don't schedule email reminders for new event types anymore on the new steps, we want to schedule them here, so no need to filter them out